.appveyor.yml

```diff
@@ -60,7 +60,7 @@ after_build:
 - ps:  clcache -z
 #- 7z a bitcoin-%APPVEYOR_BUILD_VERSION%.zip %APPVEYOR_BUILD_FOLDER%\build_msvc\%platform%\%configuration%\*.exe
 test_script:
-- cmd: src\test_bitcoin.exe -k stdout -e stdout 2> NUL
+- cmd: src\test_bitcoin.exe -l test_suite -k stdout -e stdout 2> NUL
```

Comment:
Also, it would be nice to run the tests in parallel (like we do with automake `make check -j xx`)